scripts/markdown_seo_lint.js

```diff
@@ -0,0 +1,219 @@
+const fs = require("fs");
+const yaml = require("js-yaml");
+const markdownlint = require("markdownlint");
+const path = require("path");
+
+const FRONT_MATTER_REGEX = /((^---\s*$[^]*?^---\s*$)|(^\+\+\+\s*$[^]*?^(\+\+\+|\.\.\.)\s*$))(\r\n|\r|\n|$)/m;
+
+function checkPageTitle(title) {
+    let error = null;
+    if (!title) {
+        error = "Missing page title";
+    }else if (title.trim().length === 0) {
+        error = "Page title is empty";
+    }else if (title.trim().length > 60) {
+        error = "Page title exceeds 60 characters";
+    }
+    return error;
+}
+
+function checkPageMetaDescription(meta) {
+    let error = null;
+    if (!meta) {
+        error = "Missing meta description";
+    }else if (meta.trim().length === 0) {
+        error = "Meta description is empty";
+    }else if (meta.trim().length < 50) {
+        error = "Meta description is too short. Must be at least 50 characters";
+    }else if (meta.trim().length > 160) {
+        error = "Meta descripiton is too long. Must be shorter than 160 characters";
+    }
+    return error;
+}
+
+// Recursively generate a list of markdown files from a given
+// list of paths.
```

Comment:
To be more precise, is "paths" expected to be a string array?

Also, what does this function return? What is `result` supposed to be? It seems like `options` would be a clearer name for the variable. What do you think?